(1) Community Patch/Core Files/Core Tables/CoreTableEntries.sql

```diff
@@ -323,9 +323,6 @@ ALTER TABLE Traits ADD COLUMN 'IsNoReligiousStrife' BOOLEAN DEFAULT 0;
 -- Extra Wonder Production during Golden Ages.
 ALTER TABLE Traits ADD COLUMN 'WonderProductionModGA' INTEGER DEFAULT 0;
 
--- TRAIT: Changes the food (times 100) consumed by each non-specialist citizen. --
-ALTER TABLE Traits ADD COLUMN 'NonSpecialistFoodChange' INTEGER DEFAULT 0;
-
```

Comment:
This is the trait version, not the policy one.

Reply:
> This is the trait version, not the policy one.

Fixed